cookbooks/steinberger/steinberger.prm

```diff
@@ -0,0 +1,224 @@
+  set List of model names = adiabatic, function
+
+  subsection Function
+    set Coordinate system   = spherical
+    set Variable names      = r, phi
+    set Function constants  = r0 = 3481000, r1 = 6371000
+    set Function expression = 30 * sin((r-r0)/(r1-r0)*pi)*sin(16*phi) + 20 * sin(2*(r-r0)/(r1-r0)*pi)*sin(12*phi)
+  end
+
+  subsection Adiabatic
+    set Age top boundary layer = 1e8
+    set Age bottom boundary layer = 1e8
+  end
+end
+
+# The model uses the Steinberger material model, which uses a viscosity
+# profile constrained by mineral physics and the geoid, and which can read
+# in material properties from a look-up table.
+subsection Material model
+  set Model name = Steinberger
+
+  subsection Steinberger model
+
+    # This data directory contains the files for the viscosity profile,
+    # the lateral viscosity variations due to temperature, and all
+    # material files containing look-up tables computed by mineral physics
+    # software providing density, thermal expansivity and specific heat.
+    set Data directory                                = $ASPECT_SOURCE_DIR/data/material-model/steinberger/
+    set Lateral viscosity file name                   = temp-viscosity-prefactor.txt
+    set Material file names                           = pyr-ringwood88.txt
+    set Radial viscosity file name                    = radial-visc-simple.txt
+
+    # Because of the resolution in this cookbook we limit the overall viscosity
+    # between 1e20 Pa s and 5e23 Pa s and the lateral viscosity variations to
+    # three orders of magnitude in both directions (for a total of six orders of
+    # magnitude. This allows the features of the flow field to be resolved.
+    # In the Earth, we would expect higher viscosities in the lithosphere and
+    # lower viscosities in plumes and near the core-mantle boundaries.
+    # In addition, this model only accounts for diffusion creep, so the
+    # lithosphere has a high viscosity and forms a stagnant lid on top of the
+    # model. In order to have more realistic subduction in a model like this,
+    # one would have to either prescribe plate velocities at the surface
+    # (forcing plate to subduct) or take into account plastic yielding (so that
+    # the lithosphere can break).
+    set Maximum lateral viscosity variation           = 1e3
+    set Maximum viscosity                             = 5e23
+    set Minimum viscosity                             = 1e20
+    set Reference viscosity                           = 1e22
+
+    # We base our viscosity profile on the adiabat computed based on the
+    # material porperties used in the model rather than the laterally
+    # averaged temperature.
+    set Use lateral average temperature for viscosity = false
+
+    # We want to interpolate between data points in the look-up table rather
+    # than simply taking the value of the closest point, since this is more
+    # accurate.
+    set Bilinear interpolation                        = true
```

Comment:
This is true by default anyway, can we remove these lines? Ideally I would like to remove the whole parameter and just always interpolate, I dont think there is a reason to not do it (I think I implemented this at the beginning, because I wasnt sure I implemented the interpolation correctly, yes it is that old :-)).